### PR TITLE
feat: Added new message behavior on quick order page

### LINKED ISSUE #13568: Quick order: Some questions about the behavior
**Environment Details**
Browser: Chrome 
Device: mac
Frontend: develop branch on my local
Backend: release/2011/next branch on devci4
Happens on: 20.84.65.98

**Steps**
Have 2 items in the quick order list
Try to add > than stock, so for eg, for product 3881022, add qty 60. And for product 3592865, add qty 120
You'll see that the qty has been updated to 56 and 115 respectively.
Click Add to cart.

**Actual Result**
You get an error message "Error proceeding to cart" saying the quantity was reduced. 
Screen Shot 2021-08-30 at 1 27 56 PM

**Expected Result**
Perhaps, it should be more of an information message than error message? Because the items were added in cart.

### PR DESCRIPTION
**Pull request by dydome**
close #13568

### CODE REVIEW

File: report.20210909.090326.3560.0.001.json [hunk 1]
```diff
@@ -0,0 +1,691 @@
+
```
Comment (marlass) [inline report.20210909.090326.3560.0.001.json]: This should not be commited

File: feature-libs/cart/quick-order/components/quick-order/quick-order.component.ts [hunk 2]
```diff
@@ -97,6 +109,50 @@ export class QuickOrderComponent implements OnInit, OnDestroy {
...
+      );
+      if (!element) {
+        successAddedEntries.push(entry);
+      }
+    });
+
+    this.setSuccesses(successAddedEntries);
+  }
+
+  protected clearErrorsAndWarnings(): void {
```
Comment (marlass) [inline feature-libs/cart/quick-order/components/quick-order/quick-order.component.ts]: How about changing name of this function to `clearStatuses`? As it's not only warnings and errors ;)

File: feature-libs/cart/quick-order/components/quick-order/quick-order.component.ts [hunk 3]
```diff
@@ -97,6 +109,50 @@ export class QuickOrderComponent implements OnInit, OnDestroy {
...
 
+  clearWarnings(): void {
+    this.cartWarnings$.next([]);
+  }
+
+  clearSuccesses(): void {
+    this.cartSuccesses$.next([]);
+  }
+
+  protected verifyErrors(errors: CartAddEntrySuccessEvent[]): void {
```
Comment (marlass) [inline feature-libs/cart/quick-order/components/quick-order/quick-order.component.ts]: I would change the name of all `verify*` methods to something like `extractErrors`, `extractWarnings`. That would tell more about what these functions are doing

**Review by marlass — CHANGES_REQUESTED** [review]

File: feature-libs/cart/quick-order/components/quick-order/quick-order.component.ts (hunk 3 above)
Comment (dydome) [reply feature-libs/cart/quick-order/components/quick-order/quick-order.component.ts]: good point

**Review by marlass — DISMISSED** [review]

**Review by marlass — DISMISSED** [review]

**Review by marlass — APPROVED** [review]

**Pull request merged**